subprojects/logging/src/main/java/org/gradle/internal/logging/sink/OutputEventRenderer.java

```diff
@@ -79,17 +79,36 @@ public OutputEventRenderer(final Clock clock) {
         OutputEventListener stdOutChain = new LazyListener(new Factory<OutputEventListener>() {
             @Override
             public OutputEventListener create() {
-                return onNonError(new UserInputStandardOutputRenderer(new BuildLogLevelFilterRenderer(new ProgressLogEventGenerator(new StyledTextOutputBackedRenderer(new StreamingStyledTextOutput(stdoutListeners.getSource())), false)), clock));
+                return new UserInputStandardOutputRenderer(new StyledTextOutputBackedRenderer(new StreamingStyledTextOutput(stdoutListeners.getSource())), clock);
             }
         });
-        formatters.add(stdOutChain);
         OutputEventListener stdErrChain = new LazyListener(new Factory<OutputEventListener>() {
             @Override
             public OutputEventListener create() {
-                return onError(new BuildLogLevelFilterRenderer(new ProgressLogEventGenerator(new StyledTextOutputBackedRenderer(new StreamingStyledTextOutput(stderrListeners.getSource())), false)));
+                return new StyledTextOutputBackedRenderer(new StreamingStyledTextOutput(stderrListeners.getSource()));
             }
         });
-        formatters.add(stdErrChain);
+        formatters.add(
+            new BuildLogLevelFilterRenderer(
+                new ProgressLogEventGenerator(dispatchChain(stdOutChain, stdErrChain), false)
+            )
+        );
+    }
+
+    private static OutputEventListener dispatchChain(final OutputEventListener stdoutChain, final OutputEventListener stderrChain) {
```

Comment:
Maybe bust this class out, as `OutputEventRenderer` is already pretty big.

Reply:
Done.